thirdPage.css

```diff
@@ -0,0 +1,107 @@
+body{
+    background-color: white;
+    padding:0px 40px 0px 40px;
```

Comment:
zbog ovog paddinga ti je cijeli sadrzaj sajta odvojen lijevo i desno 40px, to ne treba, nego ga pusti da se razvuce do kraja.